src/device/hub.ts

```diff
@@ -391,10 +391,18 @@ export class Hub {
         this.debugLog(`${this.device.deviceType}: ${this.accessory.displayName} updateCharacteristic CurrentTemperature: ${this.CurrentTemperature}`);
       }
     }
+
+    // CurrentAmbientLightLevel
     if (!this.device.hub?.hide_lightsensor) {
       if (this.CurrentAmbientLightLevel === undefined) {
         this.debugLog(`${this.device.deviceType}: ${this.accessory.displayName} CurrentAmbientLightLevel: ${this.CurrentAmbientLightLevel}`);
       } else {
+        if (this.device.mqttURL) {
+          mqttmessage.push(`"light": ${this.CurrentAmbientLightLevel}`);
```

Comment:
I hope "light" to be changed to "CurrentAmbientLightLevel" to keep consistency between devices.

Reply:
I am good with that.